BitwardenShared/UI/Auth/CompleteRegistration/CompleteRegistrationProcessor.swift

```diff
@@ -0,0 +1,263 @@
+                })
+                return
+            }
+            await completeRegistration()
+        }
+    }
+
+    /// Creates the user's account with their provided credentials.
+    ///
+    /// - Parameter captchaToken: The token returned when the captcha flow has completed.
+    ///
+    private func completeRegistration(captchaToken: String? = nil) async {
+        defer { coordinator.hideLoadingOverlay() }
+
+        do {
+            guard !state.passwordText.isEmpty else { throw CompleteRegistrationError.passwordEmpty }
+
+            guard state.passwordText.count >= Constants.minimumPasswordCharacters else {
+                throw CompleteRegistrationError.passwordIsTooShort
+            }
+
+            guard state.passwordText == state.retypePasswordText else {
+                throw CompleteRegistrationError.passwordsDontMatch
+            }
+
+            coordinator.showLoadingOverlay(title: Localizations.creatingAccount)
+
+            let kdf: Kdf = .pbkdf2(iterations: NonZeroU32(KdfConfig().kdfIterations))
+
+            let keys = try await services.clientService.auth().makeRegisterKeys(
+                email: state.userEmail,
+                password: state.passwordText,
+                kdf: kdf
+            )
+
+            let hashedPassword = try await services.clientService.auth().hashPassword(
+                email: state.userEmail,
+                password: state.passwordText,
+                kdfParams: kdf,
+                purpose: .serverAuthorization
+            )
+
+            _ = try await services.accountAPIService.registerFinish(
+                body: RegisterFinishRequestModel(
+                    captchaResponse: captchaToken,
+                    email: state.userEmail,
+                    emailVerificationToken: state.emailVerificationToken,
+                    kdfConfig: KdfConfig(),
+                    masterPasswordHash: hashedPassword,
+                    masterPasswordHint: state.passwordHintText,
+                    userSymmetricKey: keys.encryptedUserKey,
+                    userAsymmetricKeys: KeysRequestModel(
+                        encryptedPrivateKey: keys.keys.private,
+                        publicKey: keys.keys.public
+                    )
+                )
+            )
+
+            coordinator.navigate(to: .dismissWithAction(DismissAction {
+                self.coordinator.showToast(Localizations.accountSuccessfullyCreated)
+            }))
+        } catch let error as CompleteRegistrationError {
+            showCompleteRegistrationErrorAlert(error)
+        } catch {
+            coordinator.showAlert(.networkResponseError(error) {
+                await self.completeRegistration(captchaToken: captchaToken)
+            })
+        }
+    }
+
+    /// Sets the URLs to use.
+    ///
+    private func setRegion() async {
+        guard state.region != nil,
+              let urls = state.region?.defaultURLs else { return }
```

Comment:
⛏️ You can do all in one line:
```suggestion
        guard let urls = state.region?.defaultURLs else { return }
```